static/js/dataset-editor.js

```diff
@@ -281,7 +282,7 @@ var ClassList = React.createClass({
                 <h4>Classes</h4>
```

Comment:
Is this normal react style to add layout directly to code - or can it be extracted out into template files?